$OpenBSD: patch-libgnome-desktop_gnome-desktop-thumbnail_c,v 1.4 2011/09/26 06:20:37 ajacoutot Exp $

gnome-desktop-thumbnail.c: In function `gnome_desktop_thumbnail_factory_save_thumbnail':
gnome-desktop-thumbnail.c:1052: warning: long int format, different type arg (arg 4)
gnome-desktop-thumbnail.c: In function `gnome_desktop_thumbnail_factory_create_failed_thumbnail':
gnome-desktop-thumbnail.c:1154: warning: long int format, different type arg (arg 4)

change atol -> atoll in line with the other changes.

--- libgnome-desktop/gnome-desktop-thumbnail.c.orig	Tue Jun 14 13:32:24 2011
+++ libgnome-desktop/gnome-desktop-thumbnail.c	Mon Sep 26 07:54:00 2011
@@ -1382,7 +1382,7 @@ gnome_desktop_thumbnail_factory_save_thumbnail (GnomeD
     }
   close (tmp_fd);
   
-  g_snprintf (mtime_str, 21, "%ld",  original_mtime);
+  g_snprintf (mtime_str, 21, "%lld", (long long)original_mtime);
   width = gdk_pixbuf_get_option (thumbnail, "tEXt::Thumb::Image::Width");
   height = gdk_pixbuf_get_option (thumbnail, "tEXt::Thumb::Image::Height");
 
@@ -1484,7 +1484,7 @@ gnome_desktop_thumbnail_factory_create_failed_thumbnai
     }
   close (tmp_fd);
   
-  g_snprintf (mtime_str, 21, "%ld",  mtime);
+  g_snprintf (mtime_str, 21, "%lld", (long long)mtime);
   pixbuf = gdk_pixbuf_new (GDK_COLORSPACE_RGB, TRUE, 8, 1, 1);
   saved_ok  = gdk_pixbuf_save (pixbuf,
 			       tmp_path,
@@ -1613,7 +1613,7 @@ gnome_desktop_thumbnail_is_valid (GdkPixbuf          *
   thumb_mtime_str = gdk_pixbuf_get_option (pixbuf, "tEXt::Thumb::MTime");
   if (!thumb_mtime_str)
     return FALSE;
-  thumb_mtime = atol (thumb_mtime_str);
+  thumb_mtime = atoll (thumb_mtime_str);
   if (mtime != thumb_mtime)
     return FALSE;
   
